scripts/solution-manager.sh

```diff
@@ -0,0 +1,229 @@
+#!/bin/bash
+set -e
+set -u
+set -o pipefail
+
+SOLUTION_CONFIG="/etc/metalk8s/solutions.yaml"
+VERBOSE=${VERBOSE:-0}
+LOGFILE="/var/log/metalk8s-solution-manager.log"
```

Comment:
No related to this PR but IMO we should create a `/var/log/metalk8s/` directory as we have many log file now

Reply:
#1583